src/main/java/bio/terra/workspace/service/job/JobService.java

```diff
@@ -62,9 +63,9 @@
   private final MdcHook mdcHook;
   private final StairwayComponent stairwayComponent;
   private final FlightBeanBag flightBeanBag;
-  private FlightDebugInfo flightDebugInfo;
-
   private final Logger logger = LoggerFactory.getLogger(JobService.class);
```

Comment:
tiny nit: put `logger` back in its own block, since it's initialized differently than the other vars here?